docs/tools/vdb_table/data/neo4j.json

```diff
@@ -48,7 +49,7 @@
     "comment": ""
   },
   "multi_vec": {
-    "support": "none",
+    "support": "full",
     "source_url": "",
```

Comment:
I think users would want to see what kind of hybrid search is available in Neo4j. I wasn't able to find a source URL that describes this in the Neo4j documentation page, but I did see an example of how to use this in the LangChain [docs](https://python.langchain.com/docs/integrations/vectorstores/neo4jvector#hybrid-search-vector-keyword). Is there a source URL (or maybe even link to code on GitHub) that's from Neo4j that guides the user on learning more about hybrid search in Neo4?